lib/util/concerns/hud_lists_2024.rb

```diff
@@ -1206,10 +1211,10 @@ def data_collection_stage(id, reverse = false)
     def moving_on_assistance_options
       {
         1 => 'Subsidized housing application assistance',
-        2 => 'Financial assistance for Moving On (e.g., security deposit, moving expenses)',
-        3 => 'Non-financial assistance for Moving On (e.g., housing navigation, transition support)',
+        2 => 'Financial assistance for Moving On',
+        3 => 'Non-financial assistance for Moving On',
```

Comment:
Note to self @gigxz add this label back custom on the frontend. I think its helpful, but I can see why it might not be needed in the warehouse. @eanders

Reply:
I don't think we need it in the warehouse, but I also don't think it hurts anything if we want to keep it.